topic-operator/src/main/resources/log4j2.properties

```diff
@@ -1,15 +1,14 @@
 name = TOConfig
- 
+
 appender.console.type = Console
 appender.console.name = STDOUT
+appender.console.target = SYSTEM_OUT
```

Comment:
Are you intending to keep these changes?

Reply:
What's wrong with that? My understanding is that this is just making explicit a piece of log configuration that was implicit.

Reply:
But why do that here and not the other operators? This seems orthogonal to the rest of this refactoring.

Reply:
Ok, there is value in having the same config. Removed.